itest/map_test.go

```diff
@@ -443,6 +443,36 @@ func (ts *mapTestSuite) TestMapFromExistingByFd() {
 	ts.Equal(m1, m2)
 }
 
+func (ts *mapTestSuite) TestGetNextKey() {
+	// Create map
+	m := &goebpf.EbpfMap{
+		Type:       goebpf.MapTypeHash,
+		KeySize:    8,
+		ValueSize:  8,
+		MaxEntries: 10,
+	}
+	err := m.Create()
+	ts.NoError(err)
+
+	keys := []string{"key1", "key2", "key3"}
+
+	// Insert items into hash map
+	for index, key := range keys {
+		err = m.Insert(key, index)
+		ts.NoError(err)
+	}
+	key_to_get := ""
+	for index, k := range keys {
+		_, err := m.GetNextKey(key_to_get)
+		if index+1 == len(keys) {
+			ts.Error(err)
+		}
+		key_to_get = string(k)
+	}
```

Comment:
It is a bit unclear what this loop does.

As I understand the idea of `GetNextKey` eBPF functionality is to be able to iterate over entire hashmap (without knowing keys).
Maybe refactor this test to follow original logic? It is also could be used as good example...

Something like
```go
var currentKey string

for {
    nextKey, err := m.GetNextKey(currentKey)
    if err != nil {
        break
    }
    value, err := m.LookupInt(nextKey)
    assert.Equal(t, somethingExpected, value)

    currentKey = nextKey
}

// assert found keys to expected list
```